clock/config/database.yml

```diff
@@ -81,5 +81,5 @@ test:
 production:
   <<: *default
   database: clock_production
-  username: clock
-  password: <%= ENV['CLOCK_DATABASE_PASSWORD'] %>
+  username: peter
```

Comment:
# Here be dragons 🐉 🔥 

It's not recommended tracking this configuration file

Is a common practice just having an "example" file, like `database.yml.sample` to have an idea/draft of the structure it should follow. This way, any user is able for setting up as they want their own database.yml... BUT WHY?! 

Configurations can change btwn the user preferences, and they can change even depending on the OS they use.

In general, some configurations should not be tracked, and if possible should be delegated to the ENV VARS, for example:

For a production environment, we rather than setting here the user and the password, it's also a security concern haha

That said, Do you mind replacing the `database.yml` file with `database.yml.example` and add the first one to the `.gitignore` for avoiding other teammates tracking them.

If not, it's enough if you just undo your changes in this PR and then doing it in a new one.

Here is a resource about configuration conventions: https://12factor.net/config